src/apiCalls/transactions/sendSignedTransactions.ts

```diff
@@ -0,0 +1,22 @@
+import { Transaction } from '@elrondnetwork/erdjs';
+import axios from 'axios';
+import { networkSelector } from 'redux/selectors';
+import { store } from 'redux/store';
+
+export type SendSignedTransactionsReturnType = string[];
```

Comment:
looks like SendSignedTransactionsReturnType is just an array of strings, do we need a type for it?  let's stick with build-in types, and just use Array<strings> for now, and just use types for where there is a need for it